target/linux/ramips/patches-5.4/0008-MIPS-fix-CONFIG_MIPS_CMDLINE_DTB_EXTEND-handling

```diff
@@ -0,0 +1,32 @@
+From: Paul Cercueil <paul@xxxxxxxxxxxxxxx>
+
+commit 8e029eb0bcd6a7fab6dc9191152c085784c31ee6 upstream.
+
+The CONFIG_MIPS_CMDLINE_DTB_EXTEND option is used so that the kernel
+arguments provided in the 'bootargs' property in devicetree are extended
+with the kernel arguments provided by the bootloader.
+
+The code was broken, as it didn't actually take any of the kernel
+arguments provided in devicetree when that option was set.
+
+Fixes: 7784cac69735 ("MIPS: cmdline: Clean up boot_command_line initialization")
+Cc: stable@xxxxxxxxxxxxxxx
+Signed-off-by: Paul Cercueil <paul@xxxxxxxxxxxxxxx>
+Signed-off-by: Thomas Bogendoerfer <tsbogend@xxxxxxxxxxxxxxxx>
+Signed-off-by: Greg Kroah-Hartman <gregkh@xxxxxxxxxxxxxxxxxxx>
```

Comment:
Why are those redacted?

Reply:
Only because this is as it was in the link / source I found ([here](http://lkml.iu.edu/hypermail/linux/kernel/2003.2/01736.html)). Make sense?

Reply:
You could use this version instead:
https://git.kernel.org/linus/8e029eb0bcd6a7fab6dc9191152c085784c31ee6

Reply:
I don't mind changing it, if needed. I think the end result is the same 😄.

Thanks!